ui/lesson/ScriptingChallenge/Runner/Terminal.tsx

```diff
@@ -33,21 +32,41 @@ function Terminal({ className }: { className?: string }, ref) {
                   }
 
                   .output .error {
-                    color: #C40008;
+                    color: white;
                   }
 
-                  .output .success {
-                    color: #00B400;
+                  .success {
+                    color: white;
+                  }  
+
+                  .output .success:nth-child(3) {
+                    color: white;
+                    font-size: 18px;
+                    font-weight: 700;
+                    margin-top: 15px;
+                      }
+                     
+                  .log {
+                    color: white;
                   }
 
-                  .output .log {
+                  .output .log:first-child {
                     color:white;
+                    font-size: 18px;
+                    font-weight: 700;
+                  }
+                  
+                  .success-background {
+                    background-color: #28B12340;
+                  }
+
+                  .error-background {
+                    background-color: #D01C1640;
                   }
                 </style>
                 <div class="output"></div>
                 <script>
                   const output = document.querySelector('.output')
-
                   window.addEventListener('message', e => {
```

Comment:
no need to remove newline
```suggestion

                  window.addEventListener('message', e => {
```